chord_metadata_service/chord/workflows/wdls/cbioportal_export.wdl

```diff
@@ -87,21 +76,24 @@ task katsu_dataset_export {
 task get_maf {
     input {
         String drs_url
-        String run_dir
         String dataset_id
         String token
         Boolean validate_ssl
+        File export_data
     }
 
     command <<<
+        # Extract exported data
+        tar -xzvf ~{export_data}
```

Comment:
Maybe we should check if the extracted data exists?

Reply:
I think having this as a non-optional WDL input should (?) guarantee the path exists at least, but maybe it doesn't. It's difficult to figure out how this actually behaves.